plugins/reporters/fossid/src/test/kotlin/FossIdReporterTest.kt

```diff
@@ -85,45 +83,6 @@ class FossIdReporterTest : WordSpec({
     }
 
     "generateReport of FossIdReport " should {
-        "check if a server URL was given" {
```

Comment:
I currently don't oversee why these tests can be remove without substitute. Can you explain?

Reply:
The generated plugin factory fails if a mandatory property is missing, I don't see a need for separate tests anymore.

Reply:
Ok. I think it would be nice to mention this in the commit message.

Reply:
I already added a sentence about it.